bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -249,9 +256,6 @@ while numpy.any(fnlouder == 0):
     f['foreground/ifar'][orig_fore_idx] = sec_to_year(ifar[max_stat_idx])
     f['foreground/fap'][orig_fore_idx] = fap[max_stat_idx]
 
-    # Save the index location of the foreground trigger that was removed.
-    save_these_fore_ind = numpy.append(save_these_fore_ind, orig_fore_idx)
-
```

Comment:
Same with this block! They don't exist in pycbc master version of `pycbc_coinc_statmap`!